Webmin/input/profiles/webmin.xml

```diff
@@ -0,0 +1,43 @@
+<Profile id="Webmin" xmlns="http://checklists.nist.gov/xccdf/1.1" >
+<title>Webmin STIG</title>
```

Comment:
to play nice with our friends at DISA, consider calling this "webmin-stig-upstream" or "stig-webmin-upstream"